common/schemas/product_id_type.json

```diff
@@ -1,19 +1,6 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
-  "description": "the type of product id",
+  "description": "The UUID corresponding to the type of product id",
   "type": "string",
-  "enum": [
-    "ISBN",
-    "ISSN",
-    "ISMN",
-    "EAN",
-    "Other Standard Identifier",
-    "Standard Technical Report Number",
-    "Publisher Number",
-    "CODEN",
-    "GPO Item Number",
-    "Locally-defined identifiers",
-    "Vendor Title Number",
-    "Vendor Item Number"
-  ]
+  "pattern": "^[a-f0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$"
```

Comment:
As far as I know, we started to use GUID v4 pattern for unique ids
"pattern": "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$"

Reply:
I have updated pattern and examples as well. Thanks, @grabsefx for catching that.